scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -389,7 +389,7 @@ def generate_command(self):
             f'--build-dir={self.running_dir}',
             f'--junit-xml={self.report_file}',
             '--log-file-level=DEBUG',
-            '--log-file-format=%(asctime)s.%(msecs)d:%(levelname)s:%(name)s: %(message)s',
+            '--log-file-format=%(asctime)s.%(msecs)03d:%(levelname)s:%(name)s: %(message)s',
```

Comment:
```suggestion
            '--log-file-format=%(asctime)s,%(msecs)03d:%(levelname)s:%(name)s: %(message)s',
```
to have it the same as Twister logger does `2024-12-10 18:21:21,048`

Reply:
I would rather stick with `.`, as it is much more often used delimiter between seconds and fractions of second. You can see all the example log files in https://github.com/doublep/logview/tree/master/test, as well as `journalctl -o short-precise` (which shows microseconds instead of milliseconds, but still delimited with `.`).